modules/http/src/test/resources/disable-timeouts-config.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<mule xmlns="http://www.mulesoft.org/schema/mule/core"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:vm="http://www.mulesoft.org/schema/mule/vm"
+       xmlns:http="http://www.mulesoft.org/schema/mule/http"
+       xmlns:test="http://www.mulesoft.org/schema/mule/test"
+       xsi:schemaLocation="
+               http://www.mulesoft.org/schema/mule/core http://www.mulesoft.org/schema/mule/core/current/mule.xsd
+               http://www.mulesoft.org/schema/mule/vm http://www.mulesoft.org/schema/mule/vm/current/mule-vm.xsd
+               http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/http/current/mule-http.xsd
+               http://www.mulesoft.org/schema/mule/test http://www.mulesoft.org/schema/mule/test/current/mule-test.xsd">
+
+    <configuration defaultResponseTimeout="100"/>
+
+    <http:listener-config name="HTTP_Listener_Configuration" host="localhost" port="${port}"/>
+    <http:request-config name="HTTP_Request_Configuration" host="localhost" port="${port}"/>
+
+    <!-- HTTP Listener timeout -->
+    <flow name="HttpListenerTimeoutFlow1">
+        <vm:inbound-endpoint path="httpTimeout" exchange-pattern="request-response"/>
```

Comment:
Also, we keep away from VM as a trigger in new tests since it couples us to that connector. Use `runFlow()` instead.